### PR TITLE
Ts setup

### PR DESCRIPTION
**Pull request by nickmcmillan**
Continues https://github.com/nickmcmillan/react-physics-dragger/pull/9

### CODE REVIEW

**Comment by trblackw** [comment]
@vaibhavhrt @nickmcmillan Looks like #12 is causing some issues. Travis doesn't seem very happy with it.

**Comment by nickmcmillan** [comment]
@trblackw that's because Travis is running `lintJs` - which is no longer used. In my last commit I simply made `lintJs` an alias for `yarn run test`. It's passing now.

**Comment by trblackw** [comment]
@nickmcmillan Gotcha, looks good to me then

File: package.json [hunk 1]
```diff
@@ -13,48 +13,42 @@
     "npm": ">=5"
   },
   "scripts": {
-    "test": "cross-env CI=1 react-scripts test --env=jsdom",
-    "test:watch": "react-scripts test --env=jsdom",
+    "test": "cross-env CI=1 react-scripts-ts test --env=jsdom",
+    "test:watch": "react-scripts-ts test --env=jsdom",
+    "lintJs": "yarn run test",
```
Comment (vaibhavhrt) [inline package.json]: you can just delete this line. We can set up linter in some other PR

**Review by vaibhavhrt — COMMENTED** [review]
And remove/comment the lint script from travis.yml

**Comment by vaibhavhrt** [comment]
Looks good to me. We can decide on code style guide in some other PR and setup linting there.

**Comment by trblackw** [comment]
Agreed

**Comment by nickmcmillan** [comment]
Merged! Great work @trblackw & @vaibhavhrt. I'll release this on npm as `1.1.1`

**Comment by nickmcmillan** [comment]
Closes #5

**Comment by vaibhavhrt** [comment]
Nice, you guyz can open an issue for discussing style guide and linter to use. I am super busy these days, I will start working on writing tests first week of next month. By that time at least we will have a linter with style rules.

**Comment by trblackw** [comment]
I’m pretty busy as well, #agencylife, is there anything else that’s needed
to be done other than creating a style guide?

**Comment by vaibhavhrt** [comment]
I don't think so. Someone has asked for pagination feature but I know what the consensus is on whether to add it or not.

**Pull request merged**